data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1617 @@
+			label cutpower
+			`	Gavriil gestures at the control panel in front of him. "I do not know for sure, but based on events, I suspect that if we manually induce a power failure we should be able to reverse the process that transported us to this realm." He smiles eagerly. "I have everything all set up, and we have people waiting across the station." He looks confident. "We think that we can safely spin down the reactor and cut power across a number of these arrays in a controlled fashion. This should replicate the original power failure, except with fewer risks and less damage to the system."`
+			`	He leads you over to a panel in the main loading dock. "I know, it says 'Job Board' on it. I re-used the coding for our task list to improvise this mass-notification system." He smiles in amusement. "When you are ready to go, just select the 'De-activate Array' option from the panel. That will notify everyone and begin the shutdown." He hands you a datacrystal, and then a large case. "Here is an encrypted datacrystal of all our data so far. The case has several sets, with the data stored in a variety of different mechanisms and encryptions. We do not know what happens when the station is activated or deactivated, so multiple formats for storing the data will give a better chance that at least some of it will remain intact to be delivered." He steps back. "Well, I will return to the controls. Just give the signal when you are ready to leave. Embers light your path, Captain <first>."`
+
+
+
+mission "Remnant: Cognizance 35"
+	name "De-activate Ssil Vida Power."
+	description "Accept this to signal everyone to cut the power."
+	invisible
+	job
+	repeat
+	source "Ssil Vida"
+	to offer
+		has "Remnant: Cognizance 34: done"
+		has "Remnant: Ssil Vida Active"
+	on accept
+		event "remnant: ssil vida deactivation"
+		conversation
+			`With a shudder that is felt in every fiber of your being, you glimpse the ripple that makes reality look like the surface of a pond after a single rain drop. Almost before it begins, everything is still again. You look at your sensors, and it looks like Postverta has once again returned to its prior, barren, form.`
+			action
+				fail
+
+
+
+mission "Remnant: Cognizance 36"
+	name "Activate Ssil Vida Power."
+	description "Accept this to signal everyone to restore the power."
+	invisible
```

Comment:
```suggestion
```
Can't use the job if it's invisible!
```

Reply:
Doesn't `invisible` just control if it shows up in the active jobs list (and whether it shows the destination marker)?

I was hoping it would still be there in the job board.

Reply:
It takes up a spot on the job board, but it empty and can't be selected by the player. I think you'll just need to have it hang around in the player's log for a bit, or otherwise would have to do it through the spaceport or the like.